mu/wheels/__init__.py

```diff
@@ -19,7 +19,7 @@
 mode_packages = [
     (
         "pgzero",
-        "git+https://github.com/mu-editor/pgzero.git@mu-wheel",
+        "pgzero>=1.2.1",

```

Comment:
Minor nitpick, but we can collapse this entry in a single line now :)

Reply:
Heh.... just about to step into my penultimate meeting. Lemme see how quick I can do this... ;-)

Reply:
Turns out that's just the way black formats things..! ;-) (So no change)